docker-java-core/src/main/java/com/github/dockerjava/core/util/FiltersBuilder.java

```diff
@@ -64,6 +66,19 @@ public FiltersBuilder withContainers(String... container) {
         return getFilter("container");
     }
 
+    /**
+     * Filter by event types
+     *
+     * @param eventTypes
+     *            array of event types
```

Comment:
```suggestion
     * @param eventTypes an array of event types
```

Reply:
Oh good, I hated that ` - description` thing, but i kept it consistent.

Reply:
I don't even think it will be correctly displayed :D So let's start using the "right one", and I will fix the current ones as a follow up